configuration/external-storage.md

```diff
@@ -1,99 +1,110 @@
-# External Storage configuration
+# External storage configuration
 You can connect your external hard disk drive (HDD) to any of the USB ports on the Raspberry Pi, and mount the file system to access the data stored on your HDD. 
 
-By default, Raspberry Pi automatically mounts some of the popular file systems such as FAT, NTFS, and HFS+ at `/media/pi/<HARD-DRIVE-LABEL>` location. 
+By default, Raspberry Pi automatically mounts some of the popular file systems such as FAT, NTFS, and HFS+ at `/media/pi/<HARD-DRIVE-LABEL>` location.
```

Comment:
One for the copy-editors, but I dunno if this should be "Raspberry Pi" or "Raspbian" ?

Reply:
I will leave that for the copy editors, as I'm not sure what the convention is. It's kind of a non-issue anyway.

Reply:
If it says "Raspbian" it should probably say "Raspbian with desktop" or something as Lite doesn't mess with it.

Reply:
I don't want to make it too complicated. Raspbian with desktop is the default anyway - folk can always ask on the forums if they wonder why it's not automounted on lite.